### DISCUSSION TITLE
24-03-08 모임 불참

### DISCUSSION THREAD

**Discussion by yeslee-v** [ThinkAboutSoftware/AcademicConference#388] [General]
전 날(7일)에 여행에서 돌아오는데, 컨디션 조절을 위해 이번 주 모임은 패스하겠습니다.
더불어 책이 쉽게 안읽혀서요, 리뷰는 10일 내로 올리겠습니다.

**Comment by jongfeel** [comment]
알겠습니다.

책이 어려운 듯 하지만 이해할 수 있는 부분까지 읽는 것도 방법일 것 같습니다.

**Comment by yeslee-v** [comment]
@jongfeel 그동안 몸살로 이제서야 회복했습니다.(그래서 10일까지 PR은 올리지 못했습니다.) 해당 파트(이번 모임 범위인 10장까지)를 읽기는 다 읽었지만 정리하는게 쉽지 않네요. 이번 책은 모임에 참여하지 않고 가볍게 읽고 넘어가려고 합니다. 정안님처럼 저도 '객체지향의 사실과 오해' 타임에 돌아오겠습니다.

**Reply by jongfeel** [reply]
알겠습니다.
지난 번 모임 때도 책이 어렵다는 의견이 있어서 충분히 이해합니다.

엔터프라이즈 애플리케이션 아키텍처 패턴 마지막 모임은 5월 17일 금요일이고
객체지향 사실과 오해는 5월 18일 토요일 부터 읽기 시작하면 됩니다.

그 때쯤 되면 issues에 글이 올라올테니 확인하시고 진행하시면 됩니다. 
그리고 꼭 돌아오세요!
